src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,17 @@
+import { UserInfo } from '../UserInfo';
+
 // Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+export const TodoInfo = ({ todo }) => {
+  const { user, completed, title } = todo;
+
+  return (
+    <article className={`TodoInfo
+      ${completed ? 'TodoInfo--completed' : ''}`}
+    >
+      <h2 className="TodoInfo__title">{title}</h2>
+
+      <UserInfo user={user} key={user.id} />
```

Comment:
the user can be undefined, let's consider that case here, UserInfo doesn't need to know if the user exists, its mission is just to render it